spec/models/user_spec.rb

```diff
@@ -161,4 +163,249 @@
       end
     end
   end
+
+  describe '#visible_assessments' do
```

Comment:
These look great. Can you add one more test in for when the user is a student and the assignment is a peer review assignment.